frontend/src/components/Genre.jsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import "./Genre.css";
+
+class Genre extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
```

Comment:
Since for now you don't seem to *use* this component's `state`, I think you shouldn't have created it in the first place
This component seems like it could work as a functional one, it doesn't contain any logic yet. Do yo plan to add some here ?

Reply:
i will try to do this with a function, I made like this because: in the example Chat-app that I've got that's the used way and it worked so .. I do the same. I don't know when it's better to use state or function

Reply:
function done  ;)